upgrading/whats-new.md

```diff
@@ -1,28 +1,22 @@
 ---
 title: What's New | FusionCharts
-description: This section talks about the new features introduced in latest version.
+description: This section discusses the new features introduced in the latest version.
 heading: What's New
 ---
 
-This section is for users who are using the previous version of FusionCharts in their application. Here we'll talk about the new features introduced in v3.20.x.
+This section is for users who are using the previous version of FusionCharts in their application. Here, we'll talk about the new features introduced in v3.20.x.
 
-## Updated Support for Angular and React
 
-FusionCharts version 3.20 updates support for the `Angular`and `React Native` integrations. 
+## Update for Ramda Support
 
-FusionCharts verifies that the `Angular` integration component versions 13, 14, and 15 enable you to add JavaScript charts to your Angular application or project when using FusionCharts versions 3.20, 3.19, and 3.18. For more information, see [Angular-FusionChart](https://www.npmjs.com/package/angular-fusioncharts).
+To improve our security, FusionCharts release 3.21 updated the ´Ramda´ dependency package from version 0.25.0 to version 0.29.0. 
 
-FusionCharts verifies that the `React Native` integration component versions 0.68 and 0.69 enable you to add JavaScript charts to your React application or project when using FusionCharts versions 3.20, 3.19, and 3.18. For more information, see [React Native](https://www.npmjs.com/package/react-native-fusioncharts).
 
+## Cross-Site Scripting
 
-## Updated Support for Wrappers
+FusionCharts version 3.21 implemented the new function to eliminate any javascript functions in your code that may cause an error. The new function removes any HTML-based predefined javascript functions (like onError, onClick, Onchange, etc.) from the code and lets the charts and images display smoothly.  
```

Comment:
JavaScript not javascript(typo). :) onChange not Onchange.

Reply:
Thank you for that :)